projects/core/src/routing/routing.module.ts

```diff
@@ -54,6 +67,12 @@ export class RoutingModule {
           deps: [ConfigurableRoutesService],
           multi: true,
         },
+        {
+          provide: CONFIG_INITIALIZER,
+          useFactory: initRoutingConfig,
```

Comment:
how about rename it to `initSecurePortalConfig`?

Reply:
Done